crates/symbolicator/src/services/symbolication.rs

```diff
@@ -18,6 +18,7 @@ use sentry::protocol::SessionStatus;
 use sentry::{Hub, SentryFutureExt};
 use serde::de::DeserializeOwned;
 use serde::{Deserialize, Serialize};
+use similar::{udiff::unified_diff, Algorithm};
```

Comment:
This most definitely irrelevant nitpicking that we should not be doing.  But also rustfmt should only allow one way of formatting imports instead of tolerating so many different ways.

Anyway, I think generally we (at least I) have been trying to keep imports limited to only one level of nesting to improve readability.  In this case while there is only one level curly braces the `udiff::unified_diff` is still a different nesting level to me.

```suggestion
use similar::Algorithm;
use similar::udiff::unified_diff;
```